markdown/about.md

```diff
@@ -34,14 +35,16 @@ the nf-core alumni hall of fame:
 - [@olgabot](https://github.com/olgabot): Olga Botvinnik
 - [@sven1103](https://github.com/sven1103): Sven F.
 - [@renbot-bio](https://github.com/renbot-bio): Renuka Kudva
+- [@Xesus-Abalo](https://github.com/Xesus-Abalo): Xesús M. Abalo
 
 ## Outreach team {#outreach}
 
 The outreach team is responsible for organising hackathons, the `#bytesize` seminar series and other community outreach efforts.
-This team includes all of the `@core-team` mentioned above and also:
+This team includes all of the `@core-team` mentioned above, but mainly responsible are:
 
+- [@christopher-hakkaart](https://github.com/christopher-hakkaart): Chris Hakkaart
 - [@FranBonath](https://github.com/FranBonath): Franziska Bonath
-- [@Xesus-Abalo](https://github.com/Xesus-Abalo): Xesús M. Abalo
+- [@mribeirodantas](https://github.com/mribeirodantas): Marcel Ribeiro-Dantas
```

Comment:
Aren't both @FranBonath, @christopher-hakkaart and @mribeirodantas part of `@core-team` already?

Reply:
yes, that's why I changed the text in the outreach team paragraph. I think it is good to have official contact persons for outreach stuff.